app/src/main/java/com/medmapper/v33001/dao/DateConverter.kt

```diff
@@ -0,0 +1,17 @@
+package com.medmapper.v33001.dao
+
+import androidx.room.TypeConverter
+import java.util.Date
+
+class DateConverter {
+    @TypeConverter
+    fun fromTimestamp(value: Long?): Date? {
+        return value?.let { Date(it) }
+    }
+
+    @TypeConverter
+    fun dateToTimestamp(date: Date?): Long? {
+        return date?.time
+    }
+}
+
```

Comment:
Added the DateConverter.kt class file to enable Room to store and retrieve Date objects from the database. This ensures proper storage and retrieval of dates from the database. This is a common practice in Android development when working with Room databases. 

Source: https://developer.android.com/training/data-storage/room/referencing-data#type-converters